Habitica/src/main/java/com/habitrpg/android/habitica/helpers/notifications/HabiticaLocalNotification.java

```diff
@@ -0,0 +1,15 @@
+package com.habitrpg.android.habitica.helpers.notifications;
+
+import android.content.Context;
+
+import java.util.Map;
+
+/**
+ * Created by keithholliday on 6/28/16.
+ */
+public interface HabiticaLocalNotification {
```

Comment:
Maybe turn this into an abstract class, so that each notification doesn't have to reimplement certain code.

It could have a method that takes a NotificationCompat.Builder and then then would finish configuring that and then sending it. Each notification implementation would then configure the builder with their specific text and intents, but wouldn't have to each set the sound, icon, etc.